build/test/config.js

```diff
@@ -26,7 +26,10 @@ exports = module.exports = function(grunt, options) {
 					'http://' + host + ':<%= connect.test.options.port %>/test/commons/',
 					'http://' +
 						host +
-						':<%= connect.test.options.port %>/test/integration/rules/'
+						':<%= connect.test.options.port %>/test/integration/rules/',
+					'http://' +
+						host +
+						':<%= connect.test.options.port %>/test/integration/api/external/'
```

Comment:
I like the new directory `/api`, what is the thinking behind `/external`?

Reply:
In the future, someone may want to document the API of a submodule. It also makes it clear this only refers to the API surface area with external dependents.